index.html

```diff
@@ -233,6 +233,58 @@ <h2 class="footer-heading">🟠 Skills</h2>
       </div>
     </div>
   </section>
+  <section class="contact">
+      <h2 class="conclusion-txt">
+      I'm always interested in 
```

Comment:
- [ ] Kindly 💯 try to fix the scroll of the page because when trying to move the mouse on the page I see that there is scrolling in the page, so please try to fix that to be like the Figma design and to be great for the users 😊

> Check below to see the difference 😊⤵️
> If you can not see the image clearly just click on it to open a new tab in the browser😊

| Your design| 
| --- |
|   ![Recording 2023-02-28 at 22 46 57](https://user-images.githubusercontent.com/83360636/221976172-ee776b20-916a-4b51-88c8-f24786fd5103.gif)   |